Version 0.7.0

benmccann opened the pull request:
Release notes here: https://github.com/chartjs/chartjs-plugin-zoom/releases/edit/untagged-cef6534833172b3552b3

@simonbrunel I think you will need to create the release branch for us

simonbrunel commented:
Is it ready to be released?

simonbrunel commented on package.json:
```diff
@@ -1,7 +1,7 @@
 {
   "name": "chartjs-plugin-zoom",
   "description": "Simple HTML5 charts using the canvas element.",
```
Should we update the package description?

benmccann replied:
done

simonbrunel commented on package.json:
```diff
@@ -1,7 +1,7 @@
   "description": "Simple HTML5 charts using the canvas element.",
-  "version": "0.6.6",
+  "version": "0.7.0",
   "author": "Evert Timberg <evert.timberg@gmail.com>",
```
I would remove `author` as we did in the core repo and a few plugins

benmccann replied:
done

simonbrunel commented:
I think `chart.js` should be a peer dependency as in the [datalabels plugin](https://github.com/chartjs/chartjs-plugin-datalabels/blob/master/package.json#L54) but it may be a breaking change if anyone installed this plugin only, but not `chart.js` explicitly (thus don't have `chart.js` as dependency of their projects `package.json`).

jledentu commented:
@benmccann Don't you think that the *Breaking Changes* section should be above other sections, in the release note? These breaking changes are really important.

benmccann commented:
I moved _Breaking Changes_ to the top

I also kept Hammer.js as an external dependency and added it to the README. The thing I hadn't realized earlier is that the `main` entry point in the `package.json` file was a bundled file that included Hammer.js. This means that if a user includes Hammer.js elsewhere in their application there's no opportunity for the bundler to dedup the dependency.

Let's wait a little bit on the release to get https://github.com/chartjs/chartjs-plugin-zoom/pull/215 merged first

simonbrunel commented on README.md:
```diff
@@ -3,10 +3,16 @@
+Run `npm install --save chartjs-plugin-zoom` to install with `npm`.
+
+If including via a `<script>` tag, make sure to include `Hammer.js` as well.
+
```
I would add an example:

```
If including via a `<script>` tag, make sure to include `Hammer.js` as well:

```html
<script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
<script src="https://cdn.jsdelivr.net/npm/hammerjs@2.0.8"></script>
<script src="https://cdn.jsdelivr.net/npm/chartjs-plugin-zoom@0.7.0"></script>
```

Hammer doesn't need the extra path to the min file (`/hammer.min.js`). Until Chart.js 2.8, the path to the dist file was required (`chart.js@2.7.3/dist/Chart.min.js`) but it's not anymore the case since we changed [entry points](https://github.com/chartjs/Chart.js/blob/master/package.json#L7).

We may want the same for this plugin:

```javascript
"jsdelivr": "dist/chartjs-plugin-zoom.min.js",
"unpkg": "dist/chartjs-plugin-zoom.min.js",
"main": "dist/chartjs-plugin-zoom.js",
```

simonbrunel commented on README.md:
```diff
@@ -3,10 +3,16 @@
 
 Panning can be done via the mouse or with a finger.
-Zooming is done via the mouse wheel or via a pinch gesture. [Hammer JS](http://hammerjs.github.io/) is used for gesture recognition.
+Zooming is done via the mouse wheel or via a pinch gesture. [Hammer.js](http://hammerjs.github.io/) is used for gesture recognition.
```
`https://hammerjs....`

benmccann replied:
done. updated everywhere else as well

simonbrunel commented on samples/pan-bar.html:
```diff
@@ -3,9 +3,9 @@
-	<script src="../node_modules/hammerjs/hammer.min.js"></script>
-	<script src="../dist/chartjs-plugin-zoom.js"></script>
+	<script src="https://cdn.jsdelivr.net/npm/chart.js@2.7.3/dist/Chart.min.js"></script>
+	<script src="https://cdn.jsdelivr.net/npm/hammerjs@2.0.8/hammer.min.js"></script>
```
That are the links I sent you on Slack but actually we don't need the extra file path for Hammer. It will be the same with Chart.js starting v2.8, so this would become:

```html
<script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
<script src="https://cdn.jsdelivr.net/npm/hammerjs@2.0.8"></script>
<script src="../dist/chartjs-plugin-zoom.min.js"></script>
```

benmccann commented on package.json:
```diff
@@ -29,8 +28,10 @@
+  },
   "dependencies": {
-    "chart.js": "^2.6.0",
     "hammerjs": "^2.0.8"
```
The goal was to have it specified as a dependency so that npm would pull it in. We had to make it external so that it wouldn't be bundled and pulled in via npm. I think npm should manage the dependency so that if another dependency includes hammer it can be deduped. I don't think the user should have to specify hammer directly in their project though. So I do think `dependencies` is probably the right place to specify it
